content/src/service/snapshots/SnapshotManager.ts

```diff
@@ -166,11 +183,20 @@ export class SnapshotManager {
 
         // add the entoty to the inMemoryArray to be used by the legacy formatter
         appendToInMemoryArray(snapshotElem.entityType as EntityType, [snapshotElem.entityId, snapshotElem.pointers])
+
+        if (newActiveEntitiesCount[snapshotElem.entityType] != null) {
+          newActiveEntitiesCount[snapshotElem.entityType]++
+        } else {
+          newActiveEntitiesCount[snapshotElem.entityType] = 1
+        }
```

Comment:
This is duplicated logic from the line before it

```ts
appendToInMemoryArray(snapshotElem.entityType as EntityType, [snapshotElem.entityId, snapshotElem.pointers])
```

After this for-each it can be inferred from the in-memory-arrays

```ts
   const inMemoryArrays: Partial<Record<EntityType, Array<[string, string[]]>>> = {}
```

@jmoguilevsky